conf/edi.conf.part

```diff
@@ -79,6 +79,45 @@ source src_ch_bfs_gebaeude_wohnungs_register : def_searchable_features
         FROM bfs.gwr_sphinx
 }
 
+
+source src_ch_bak_bundesinventar_schuetzenswerte_ortsbilder : def_searchable_features
+{
+    sql_db = bak_dev
+    sql_query = \
+        SELECT row_number() OVER(ORDER BY xtf_id asc) as id \
+            , name as label \
+            , 'feature' as origin \
+            , remove_accents(concat_ws(' ',id::text,name)) as detail \
+            , 'ch.bak.bundesinventar-schuetzenswerte-ortsbilder' as layer \
+            , quadindex(the_geom) as geom_quadindex \
+            , st_y(st_transform(st_centroid(the_geom),4326)) as lat \
+            , st_x(st_transform(st_centroid(the_geom),4326)) as lon \
+            , box2d(st_transform(the_geom, 21781)) as geom_st_box2d \
+            , box2d(st_transform(the_geom, 2056)) as geom_st_box2d_lv95 \
+            , xtf_id::text as feature_id \
+        from isos_ortsbild
+}
+
+
+source src_ch_bak_bundesinventar_schuetzenswerte_ortsbilder_fotos : def_searchable_features
+{
+    sql_db = bak_dev
+    sql_query = \
+        SELECT row_number() OVER(ORDER BY xtf_id asc) as id \
+            , name as label \
+            , 'feature' as origin \
+            , remove_accents(concat_ws(' ',id::text,name)) as detail \
+            , 'ch.bak.bundesinventar-schuetzenswerte-ortsbilder_fotos' as layer \
+            , quadindex(the_geom) as geom_quadindex \
+            , st_y(st_transform(st_centroid(the_geom),4326)) as lat \
+            , st_x(st_transform(st_centroid(the_geom),4326)) as lon \
+            , box2d(st_transform(the_geom, 21781)) as geom_st_box2d \
+            , box2d(st_transform(the_geom, 2056)) as geom_st_box2d_lv95 \
+            , xtf_id::text as feature_id \
+        from ( SELECT * FROM view_isos_foto UNION SELECT * FROM view_isos_foto_ob UNION SELECT * FROM view_isos_foto_obt UNION SELECT * FROM view_isos_foto_obth) as f
```

Comment:
@ltclm what about putting the logic into a view?

Reply:
yes thats a good idea!

Reply:
Done!